app/about_us/OurCrew.js

```diff
@@ -0,0 +1,68 @@
+import styles from './page.module.css';
+
+export const OurCrew = () => {
+    return (
+        <section>
+            <p className={styles.textCrew}>Our crew is the heart and soul of Galactica. We are a diverse team of
+                seasoned space explorers,
+                engineers, and visionaries who are united by a common goal: to make space travel accessible and exciting
+                for all.</p>
+            <div className={styles.containerCrew}>
```

Comment:
Instead of repeating similar JSX for each crew member, use an array of crew members and map over it. This reduces redundancy and makes it easier to add or remove members.
like this : {crewMembers.map((member, index) => (....)